src/themes/theme.js

```diff
@@ -0,0 +1,38 @@
+import tw from "twin.macro";
+
+const theme = {
+  button: {
+    default: {
+      base: tw`inline-flex items-center shadow-sm font-medium`,
+      color: tw`text-white-100 bg-indigo-600`,
+      hover: tw`hover:bg-indigo-700`,
+      focus: tw`focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500`,
+    },
+    color: {
+      primary: tw`bg-primary-500 hover:bg-primary-600 border-primary-500`,
+      success: tw`bg-success-500 hover:bg-success-600 border-success-500`,
+      danger: tw`bg-danger-600 hover:bg-danger-700 border-danger-600`,
+    },
+    border: {
+      gray: tw`border border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
+      grey: tw`border border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
+      transparent: tw`bg-transparent`,
+      primary: tw`border-primary-500 hover:bg-primary-500 hover:text-gray-50`,
+      success: tw`border-success-500 hover:bg-success-500 hover:text-gray-50`,
+      danger: tw`border-danger-500 hover:bg-danger-500 hover:text-gray-50`,
+    },
```

Comment:
Let's rework the code as follows:

    border: {
      base:  tw`border`,
      gray: tw`border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
      grey: tw`border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
      transparent: tw`bg-transparent`,
      primary: tw`border-primary-500 hover:bg-primary-500 hover:text-gray-50`,
      success: tw`border-success-500 hover:bg-success-500 hover:text-gray-50`,
      danger: tw`border-danger-500 hover:bg-danger-500 hover:text-gray-50`,
    },

Reply:
fixed